### PR TITLE
Create it.po

### PR DESCRIPTION
**Pull request by albanobattistella**

### CODE REVIEW

File: po/it.po [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: 1.0.1\n"
+"Report-Msgid-Bugs-To: Jean Lorenz Hertel <jean.hertel@hotmail.com>\n"
+"POT-Creation-Date: 2018-08-06 08:10-0300\n"
+"PO-Revision-Date: 2020-01-31 19:10-0100\n"
+"Last-Translator: albano battistella <albano_battistella@hotmail.com>\n"
+"Language-Team: Italian <albano_battistella@hotmail.com>\n"
+"Language: \n"
```
Comment (jlHertel) [inline po/it.po]: I think you are missing the language code here.
Apart from that, the translation seems awesome.

**Comment by albanobattistella** [comment]
Hi,

done.

albano

________________________________
Da: Jean Lorenz Hertel <notifications@github.com>
Inviato: sabato 1 febbraio 2020 10:32
A: jlHertel/adriconf <adriconf@noreply.github.com>
Cc: albanobattistella <albano_battistella@hotmail.com>; Author <author@noreply.github.com>
Oggetto: Re: [jlHertel/adriconf] Create it.po (#80)


@jlHertel requested changes on this pull request.

Can you also add your name to the AUTHORS file?

________________________________

In po/it.po<https://github.com/jlHertel/adriconf/pull/80#discussion_r373769061>:

> @@ -0,0 +1,322 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# ALBANO BATTISTELLA <albano_battistella@hotmail.com>, 2020.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: 1.0.1\n"
+"Report-Msgid-Bugs-To: Jean Lorenz Hertel <jean.hertel@hotmail.com>\n"
+"POT-Creation-Date: 2018-08-06 08:10-0300\n"
+"PO-Revision-Date: 2020-01-31 19:10-0100\n"
+"Last-Translator: albano battistella <albano_battistella@hotmail.com>\n"
+"Language-Team: Italian <albano_battistella@hotmail.com>\n"
+"Language: \n"


I think you are missing the language code here.
Apart from that, the translation seems awesome.

**Comment by jlHertel** [comment]
One week has passed and the issue hasn't been addressed.
@albanobattistella You still want to add this?
If yes, please add the proper language code to the translation, as I indicated on my previous comment on the code.

**Comment by albanobattistella** [comment]
Hi,

added.


________________________________
Da: Jean Lorenz Hertel <notifications@github.com>
Inviato: venerdì 7 febbraio 2020 21:09
A: jlHertel/adriconf <adriconf@noreply.github.com>
Cc: albanobattistella <albano_battistella@hotmail.com>; Mention <mention@noreply.github.com>
Oggetto: Re: [jlHertel/adriconf] Create it.po (#80)


One week has passed and the issue hasn't been addressed.
@albanobattistella<https://github.com/albanobattistella> You still want to add this?
If yes, please add the proper language code to the translation, as I indicated on my previous comment on the code.

**Pull request merged**